dbt/include/firebolt/macros/adapters.sql

```diff
@@ -184,32 +195,42 @@
 {% endmacro %}
 
 
-{% macro firebolt__create_table_as(temporary, relation, sql) -%}
+{% macro firebolt__create_table_as(temporary, relation, ctas_sql) -%}
```

Comment:
Why rename this to ctas_sql? The argument is a select statement and is generating CTAS

Reply:
I did this for future-me, because it was unclear what exactly the sql was or where it was coming from. It is, technically, generating just a SELECT query, but that query is used in this macro specifically for creating a table using CTAS. I'll change it back if you prefer, or if you think it would be more technically correct. Or I could call it query_sql or ctas_query_sql.

Reply:
I don't think it _really_ matters, but I think it could be a little confusing for new contributors. What about `model_sql` since its coming from the dbt model? Or if you don't mind absurdly verbose `select_sql_to_be_used_in_the_as_clause_of_create_table_as`

Reply:
Having had to deal a lot with other peoples' code, I actually prefer verbose. We know we're in the `create_table_as` macro though, so how about just `select_sql_to_be_used_in_the_as_clause`?

Reply:
Since you have a clear description of the arg in the docstring, I think `select_sql` is probably sufficient

Reply:
I was actually thinking that, so I'm glad we're on the same page. Still, lol.